### ISSUE TITLE
[bug]: Flag --gui no longer working

### ISSUE THREAD

**Issue by dhelonious** [invoke-ai/InvokeAI#1870]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

8GB

### What happened?

Calling `invoke.py --gui` gives the following error:

```
TypeError: FlaskUI.__init__() got an unexpected keyword argument 'start_server'
```

This seems to be an issue with newer versions of `flaskwebgui`. Apparently, the `start_server` argument was renamed to `server` in [commit e023f1c](https://github.com/ClimenteA/flaskwebgui/commit/e023f1cd8627591be0cfed014b8fafa9610362d8). However, also the `idle_interval` argument is no longer present in current releases of `flaskwebgui`.

I noticed that no version of `flaskebgui` is specified in the `environment.yaml` (in fact, `flaskwebgui` is not even included). Adding `flaskwebgui==0.3.7` would also solve this issue.

**Comment by addianto** [comment]
Previously in #1687, I changed `flaskwebui` initialization process in order to fix similar issue that happend in InvokeAI that was installed manually using `pip`. I was not aware with the version difference of `flaskwebgui` in the requirements file used by `pip` and `conda` installers.

I think this issue can be fixed quickly if the project still want to maintain `--gui` option. As @dhelonious said, `flaskwebgui` version needs to be pinned in the requirements files used by `pip` and `conda`.

**Comment by dhelonious** [comment]
@ebr: I actually like to have a separate window for the UI. I would not mind using the `--web` flag if there was an option to automatically open the url in my browser. However, I'm also fine if the maintainers decide to drop the `--gui` flag, as long as there is no longer a broken flag.

**Comment by blessedcoolant** [comment]
Didnt decide to drop it. The library was updated and it the new version has a different syntax. I think we didnt pin the version in the installer.

As a temporary fix you can do this.

Go to `invoke_ai_web_server.py` and find this code and replace it to the one below.

From:

```python
FlaskUI(
                    app=self.app,
                    socketio=self.socketio,
                    start_server="flask-socketio",
                    width=1600,
                    height=1000,
                    idle_interval=10,
                    port=self.port
                ).run()
```

To:

```python
FlaskUI(
                    app=self.app,
                    socketio=self.socketio,
                    server="flask_socketio",
                    width=1600,
                    height=1000,
                    port=self.port
                ).run()
```

**Comment by dhelonious** [comment]
Thanks, I'll look forward to the update! As a workaround, I downgraded `pywebgui` in my virtual env, so that I do not have to make changes that could be overwritten on `git pull`.

**Comment by blessedcoolant** [comment]
> Thanks, I'll look forward to the update! As a workaround, I downgraded `pywebgui` in my virtual env, so that I do not have to make changes that could be overwritten on `git pull`.

`flaskwebgui` .. Not pywebgui

**Comment by dhelonious** [comment]
Typo... thanks! I corrected this.

**Comment by dhelonious** [comment]
Fixed in 2.2.4
